notNeededPackages.json

```diff
@@ -312,6 +312,26 @@
             "libraryName": "badgen",
             "asOfVersion": "2.7.1"
         },
+        "barnard59": {
+            "libraryName": "barnard59",
+            "asOfVersion": "4.4.0"
+        },
+        "barnard59-base": {
+            "libraryName": "barnard59-base",
+            "asOfVersion": "2.3.0"
+        },
+        "barnard59-core": {
+            "libraryName": "barnard59-core",
+            "asOfVersion": "5.3.0"
+        },
+        "barnard59-graph-store": {
+            "libraryName": "barnard59-graph-store",
+            "asOfVersion": "5.1.0"
+        },
+        "barnard59-validate-shacl": {
+            "libraryName": "barnard59-shacl",
+            "asOfVersion": "1.3.0"
+        },
```

Comment:
These changes should only be here if the upstream package has added types; it sounds like that is not the case, so this should be reverted.

Reply:
Ah, sorry, I see, most of them are typed, but `barnard59-validate-shacl` is not, and won't because it's been renamed.

That one should be reverted as upstream isn't typed, I believe...

Reply:
Yeah, I'm fairly certain that the libraryName should not differ from the actual name of the library.

Reply:
> Yeah, I'm fairly certain that the libraryName should not differ from the actual name of the library.

Well, it's actually a case mentioned in the readme:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md?plain=1#L209

Reply:
Huh, weird, my memory is off I suppose. @sandersn @andrewbranch probably know how this works better than me but it didn't seem like it would actually work from my skimming of our code.

Reply:
The jsdoc for that property says "The npm name of the package the types package was for." Let me check the code: if it generates a dependency on barnard59-shacl and a message that says "please use barnard59-shacl", that's good enough, right?

Later: looking at createNotNeededPackageJson, it publishes `pkg.name`, which should be types/barnard59-validate-shacl, but adds a dependency on libraryName, which should be barnard59-shacl. And the message references barnard59-shacl. Seems right to me.

Reply:
Ao this is correct right ?